custom/icds_reports/reports/awc_reports.py

```diff
@@ -1176,20 +1177,22 @@ def get_pregnant_details(case_id, awc_id):
         if row_data['trimester'] >= current_trimester:
             config['data'][row_data['trimester'] - 1].append(dict(
                 case_id=row_data['case_id'],
-                trimester=row_data['trimester'],
-                person_name=row_data['person_name'],
+                trimester=row_data['trimester'] if row_data['trimester'] else DATA_NOT_ENTERED,
+                person_name=row_data['person_name'] if row_data['person_name'] else DATA_NOT_ENTERED,
                 age=row_data['age_in_months'] // 12 if row_data['age_in_months'] else row_data['age_in_months'],
-                mobile_number=row_data['mobile_number'],
-                edd=row_data['edd'],
-                opened_on=row_data['opened_on'],
-                preg_order=row_data['preg_order'],
-                home_visit_date=row_data['home_visit_date'],
+                mobile_number=row_data['mobile_number'] if row_data['mobile_number'] else DATA_NOT_ENTERED,
+                edd=row_data['edd'] if row_data['edd'] else DATA_NOT_ENTERED,
+                opened_on=row_data['opened_on'] if row_data['opened_on'] else DATA_NOT_ENTERED,
+                preg_order=row_data['preg_order'] if row_data['preg_order'] else DATA_NOT_ENTERED,
+                home_visit_date=row_data['home_visit_date'] if row_data['home_visit_date'] else DATA_NOT_ENTERED,
                 bp='{} / {}'.format(
-                    row_data['bp_sys'] if row_data['bp_sys'] else '--',
-                    row_data['bp_dia'] if row_data['bp_dia'] else '--',
+                    row_data['bp_sys'] if row_data['bp_sys'] else DATA_NOT_ENTERED,
+                    row_data['bp_dia'] if row_data['bp_dia'] else DATA_NOT_ENTERED,
```

Comment:
I think if both are `None` this should show `Data Not Entered` not `Data Not Entered / Data Not Entered`